gulp/parentAliases.js

```diff
@@ -11,12 +11,13 @@ const traverseAliases = themePath => {
     );
 
     const parentMatch = themeXML.match(
-        /<parent>[a-z]+\/[^\-]+\-([a-z]+)<\/parent>/i
+        /<parent>([a-z]+)\/[^\-]+\-([a-z]+)<\/parent>/i
     );
 
     if (parentMatch) {
-        const parentName = parentMatch[1];
-        const parentPath = path.resolve(`../theme-${parentName}`);
+        const parentVendor = parentMatch[1]
```

Comment:
Hello @marcoveeneman 
First of all, thank you for preparing this MR. 

Parent theme vendor is defined in theme.xml in uppercase, like
` <parent>Testcompany/theme-test</parent>` 

in this case, the path won't be correctly resolved. 

Please just convert parentVendor to lowercase and it should solve the problem. Still, the solution won't cover vendors that have multi-part names (like test-company), but in all the other cases it helps. Thanks.

Reply:
Hi @drabikowy, thanks for the quick reply!

Ok, i think i see the problem, all directories in vendor are lowercase, but using the current patch the parentVendor can contain capitals. I'll convert it to lowercase as you suggested.